Documentation/ApiOverview/Database/RestrictionBuilder/Index.rst

```diff
@@ -223,3 +228,51 @@ deliver and use an own set of restrictions for own query statements if needed.
    It can be very helpful to debug the final statements created by the `RestrictionBuilder` using
    :php:`debug($queryBuilder->getSQL())` right before the final call to :php:`$queryBuilder->execute()`. Just
    take care these calls **do not** :ref:`end up in production <database-query-builder-get-sql>` code.
+
+
+A very complex example is that you might want to apply one or more restriction/s to only one table. Let's say,
+that you have content in table `tt_content` that could have another record as parent record assigned in field
+`parent`. Now you would like to get all records except the child records where the parent record is hidden.
+
+   // use TYPO3\CMS\Core\Utility\GeneralUtility;
```

Comment:
This should be a code block. Just add a :: to the end of the previous sentence, e.g. 

> where the parent record is hidden::

This is the same as doing:

> where the parent record is hidden:
> .. code-block:: php

(because our default language for code blocks is defined as PHP in TYPO3 Explained)

This is a .rst gotcha because you do not need to do that in Markdown.